MapboxNavigation/RouteTableViewController.swift

```diff
@@ -43,7 +43,7 @@ class RouteTableViewController: UIViewController {
         }
         
         if routeProgress.durationRemaining < 60 {
-            headerView.timeRemaining.text = String.localizedStringWithFormat(NSLocalizedString("LESS_THAN", bundle: .mapboxNavigation, value: "<%@", comment: "Format string for less than; 1 = duration remaining"), dateComponentsFormatter.string(from: 61)!)
+            headerView.timeRemaining.text = String.localizedStringWithFormat(NSLocalizedString("LESS_THAN", bundle: .mapboxNavigation, value: "%@", comment: "Format string for less than; 1 = duration remaining"), dateComponentsFormatter.string(from: 61)!)
```

Comment:
Why was the less-than sign removed? The label was intended to say for instance “<1 s`. If this removal was intentional, then there’s no point to a format string that consists of nothing but a single placeholder.